src/modules/Builder/features/ActivityItemsFlow/ItemFlow/ConditionRow/ConditionRow.utils.ts

```diff
@@ -0,0 +1,98 @@
+import i18n from 'i18n';
+import { ItemFormValues } from 'modules/Builder/pages';
+import { ItemResponseType, ConditionType } from 'shared/consts';
+import { getEntityKey } from 'shared/utils';
+import {
+  Condition,
+  OptionCondition,
+  SingleValueCondition,
+  RangeValueCondition,
+  SingleAndMultipleSelectItemResponseValues,
+} from 'shared/state';
+import { ConditionItemType } from 'modules/Builder/components';
+
+import { DEFAULT_PAYLOAD_MIN_VALUE, DEFAULT_PAYLOAD_MAX_VALUE } from './ConditionRow.const';
+
+const { t } = i18n;
+
+export const getConditionItemType = (item: ItemFormValues) => {
+  switch (item.responseType) {
+    case ItemResponseType.Slider:
+      return ConditionItemType.Slider;
+    case ItemResponseType.SingleSelection:
+      return ConditionItemType.SingleSelection;
+    case ItemResponseType.MultipleSelection:
+      return ConditionItemType.MultiSelection;
+    default:
+      return ConditionItemType.SingleSelection;
+  }
+};
+
+export const getItemOptions = (items: ItemFormValues[]) =>
+  items?.reduce(
+    (optionList: { labelKey: string; value: string; type: ConditionItemType }[], item) => {
```

Comment:
pls move type of optionList to ConditionRow.types.ts

Reply:
done